playbooks/installer/ospd/upgrade/7_3-8_0.yml

```diff
@@ -0,0 +1,141 @@
+---
+- name: Prepare the undercloud
+  hosts: undercloud
+  gather_facts: no
+  tasks:
+    - name: Setting up node
+      yum: name=http://sat618-qe-rhel7.usersys.redhat.com/pub/katello-ca-consumer-latest.noarch.rpm state=present
```

Comment:
Note, any internal URLs should be in private.yml, not explicit here.
Also, format:
`yum:
    name: <URL>
    state: present
`